src/backend/downloadmanager/utils.ts

```diff
@@ -185,7 +185,7 @@ async function downloadFixesFor(appName: string, runner: Runner) {
   if (!existsSync(fixesPath)) {
     mkdirSync(fixesPath, { recursive: true })
   }
-  downloadFile({ url, dest })
+  downloadFile({ url, dest, ignoreFailure: false })
```

Comment:
Shouldn't this be
```suggestion
  downloadFile({ url, dest, ignoreFailure: true })
```
now, to silence the error?

Reply:
oooop, yes